pandas/core/indexes/timedeltas.py

```diff
@@ -166,6 +167,19 @@ def __new__(cls, data=None, unit=None, freq=None, start=None, end=None,
         elif copy:
             data = np.array(data, copy=True)
 
+        data = np.array(data, copy=False)
```

Comment:
do you need this here? the array conversion

Reply:
This whole block is taken out of the top of `TimedeltaArray._simple_new`.  I think we could get here with e.g. another TimedeltaIndex for `data`.